src/mono/wasm/debugger/BrowserDebugProxy/DebuggerProxy.cs

```diff
@@ -16,9 +17,9 @@ public class DebuggerProxy
     {
         private readonly MonoProxy proxy;
 
-        public DebuggerProxy(ILoggerFactory loggerFactory)
+        public DebuggerProxy(ILoggerFactory loggerFactory, StringValues urlSymbolServerList)
```

Comment:
Should we maybe take a parameter of type `IEnumerable<string>` instead, or `IList<string>`? And same for `MonoProxy`.